elements/lisk-p2p/src/peer/base.ts

```diff
@@ -151,11 +149,8 @@ export class Peer extends EventEmitter {
 		this._peerConfig = peerConfig;
 		this._ipAddress = peerInfo.ipAddress;
 		this._wsPort = peerInfo.wsPort;
-		this._id = constructPeerIdFromPeerInfo({
-			ipAddress: this._ipAddress,
-			wsPort: this._wsPort,
-		});
-		this._height = peerInfo.height ? (peerInfo.height as number) : 0;
+		this._id = peerInfo.peerId;
```

Comment:
Shouldn't this come from `this._peerInfo`? Assess if this can be tackle here or we should open a specific issue.

Reply:
created #4401 issue 👆